corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -625,6 +625,15 @@ def render(self, context):
     return AddToBlockNode(nodelist, 'js-inline')
 
 
+@register.simple_tag
+def html_attr(value):
+    if isinstance(value, basestring):
+        value = json.dumps(value)[1:-1]
```

Comment:
Okay, I squinted at this for a bit and concluded that there's no reason to call `json.dumps` when the given value is just a string, which simplifies the code and takes care of the backslash issue. Added a commit.